src/runner/TestRunner.ts

```diff
@@ -38,6 +38,8 @@ export class TestRunner {
         testFiles: string[]
     ) {
         testFiles.forEach((filename, index) => {
+            // Don't allow test files to pollute each other
+            _brs_.resetMocks();
```

Comment:
I don't think this will quite work -- `_brs_` is only defined in Brightscript files, not Typescript, which is why we have compilation errors from Github Actions. I think we'll probably have to expose `resetMocks` as an export in the `brs` project, then call it from here.